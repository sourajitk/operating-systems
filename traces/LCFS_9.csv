1,0,1
2,0,82
3,0,83
4,0,84
5,0,85
6,0,86
7,0,87
8,0,88
9,0,89
10,0,90
11,0,91
12,0,92
13,0,93
14,0,94
15,0,95
16,0,96
17,0,97
18,0,98
19,0,99
20,0,100
