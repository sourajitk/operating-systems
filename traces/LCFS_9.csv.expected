1,1
2,1730
3,1648
4,1565
5,1481
6,1396
7,1310
8,1223
9,1135
10,1046
11,956
12,865
13,773
14,680
15,586
16,491
17,395
18,298
19,200
20,101
